README.md

```diff
@@ -317,6 +317,45 @@ now keep this in mind:
 The CLI text editor we all know, just in case you need to inspect some bug in
 hot deployments.
 
+### `install-addons`
+
+This script will iterate the `addons.yml` file to gather support addons, and
+subsequently install to the Odoo instance. It will then exit.
+
+This script is great to use during buildout in order to prepare an instance.
+Below is a minimalistic `docker-compose` file illustrating a use case:
+
+```yaml
+version: '2'
+
+services:
+
+  install:
+    image: laslabs/odoo-buildouts:dev-10
+    command: 'install-addons'
+    tty: true
+    depends_on:
+      - db
+    tty: true
+
+  web:
+    image: laslabs/odoo-buildouts:dev-10
+    restart: unless-stopped
+    tty: true
+    depends_on:
+      - db
+      - install
+    ports:
+      - 8069
+      - 8071
+    environment:
+      WAIT_NOHOST: 'install'
+
+  db:
+    image: postgres:9.6-alpine
```

Comment:
I feel like this adds complexity that is not required for an example. For me- the environment variables that are present on build took a long time to understand in terms of what needs to be present at each build step, which only got confused further when I started thinking mass distribution of images.

For example, my `docker-compose.yml` for builds look like this:

```yml
version: "2.1"

services:
  odoo:
    image: lasley/odoo-buildouts:smd-10
    build:
      context: ./odoo
```

Really it could be a simple `docker build` if I wasn't lazy and didn't want to have to remember the image name. 

I digress though, I think static version is better here.